test/e2e/cluster_upgrade.go

```diff
@@ -30,6 +30,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/ginkgowrapper"
 	"k8s.io/kubernetes/test/e2e/upgrades"
+	apps "k8s.io/kubernetes/test/e2e/upgrades/apps"
```

Comment:
i see. i will look into the BUILD files and run `update/bazel.sh` to generate the correct binaries.